portfolio/src/main/webapp/js/script.js

```diff
@@ -11,19 +11,44 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+var mode = false;
 
 /**
- * Switches Light Mode of Page
+ * Sets Light mode of page based on cookies
  */
-function switchMode() {
-  var style = document.getElementById('pagestyle').getAttribute('href');
-  if (style == '') { 
+function setMode() { 
+  var value = getCookie('mode');
+  if(value=="dark"){
     switchSheet('css/nightmode.css');
     switchIcon('fas fa-sun');
-  } else { 
+    mode = true;
+  }else {
     switchSheet('');
     switchIcon('fas fa-moon');
+    mode = false;
+    }
+}
+
+/**
+ * @param {string} The string whose value you want to get from the cookie. 
+ */
+function getCookie(name) { 
+  var re = new RegExp(name + "=([^;]+)");
```

Comment:
use `const` (or `let` when appropriate) instead of var

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let